docs/sources/operations/storage/wal.md

```diff
@@ -95,13 +95,13 @@ After hitting the endpoint for `ingester-2 ingester-3`, scale down the ingesters
 
 Statefulsets are significantly more cumbersome to work with, upgrade, and so on. Much of this stems from immutable fields on the specification. For example, if one wants to start using the WAL with single store Loki and wants separate volume mounts for the WAL and the boltdb-shipper, you may see immutability errors when attempting updates the Kubernetes statefulsets.
 
-In this case, try `kubectl -n <namespace> delete sts ingester --cascade=false`. This will leave the pods alive but delete the statefulset. Then you may recreate the (updated) statefulset and one-by-one start deleting the `ingester-0` through `ingester-n` pods _in that order_, allowing the statefulset to spin up new pods to replace them.
+In this case, try `kubectl -n <namespace> delete sts ingester --cascade=false`. This will leave the pods alive but delete the StatefulSet. Then you may recreate the (updated) StatefulSet and one-by-one start deleting the `ingester-0` through `ingester-n` pods _in that order_, allowing the StatefulSet to spin up new pods to replace them.
```

Comment:
Fixing Pod whilst we are here.

```suggestion
In this case, try `kubectl -n <namespace> delete sts ingester --cascade=false`.
This will leave the Pods alive but delete the StatefulSet.
Then you may recreate the (updated) StatefulSet and one-by-one start deleting the `ingester-0` through `ingester-n` Pods _in that order_, allowing the StatefulSet to spin up new pods to replace them.
```